src/Layer/Launchpad/Launchpad.lua

```diff
@@ -118,9 +118,30 @@ function Launchpad:send(channel, number, value)
     if (not self.midi_out or not self.midi_out.is_open) then
         return
     end
-    local message = {channel, number, value}
-    -- print(("Launchpad : send MIDI %X %X %X"):format(message[1], message[2], message[3]))
-    self.midi_out:send(message)
+	if (channel == 0x90 and value > 127) then
```

Comment:
The new velocities MORE OR LESS map the same colors as the original
launchpad.
even here, novation decided to make things completely incompatible....

Il domenica 28 febbraio 2016, Ingolf Wagner <notifications@github.com> ha
scritto:

> In src/Layer/Launchpad/Launchpad.lua
> <https://github.com/mrVanDalo/stepp0r/pull/90#discussion_r54349406>:
>
> > @@ -118,9 +118,30 @@ function Launchpad:send(channel, number, value)
> >      if (not self.midi_out or not self.midi_out.is_open) then
> >          return
> >      end
> > -    local message = {channel, number, value}
> > -    -- print(("Launchpad : send MIDI %X %X %X"):format(message[1], message[2], message[3]))
> > -    self.midi_out:send(message)
> > +	if (channel == 0x90 and value > 127) then
>
> Ok, I see! but all other messages are handled like before, just with
> different velocities, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrVanDalo/stepp0r/pull/90/files#r54349406>.
>